esmvalcore/preprocessor/_regrid.py

```diff
@@ -302,6 +306,43 @@ def _attempt_irregular_regridding(cube, scheme):
     return False
 
 
+def extract_location(cube, location, scheme):
+    """Extract a point using a location name, with interpolation.
+
+    Extracts a single location point from a cube, according
+    to the interpolation scheme `scheme`.
+
+    The function just retrieves the coordinates of the location and then calls
+    the `extract_point` preprocessor.
+
+    It can be use to locate cities and villages, but also mountains or other
+    geographical locations.
+
+    Parameters
+    ----------
+    cube : cube
+        The source cube to extract a point from.
+
+    location : str
+        The reference location.
+
+    scheme : str
+        The interpolation scheme. 'linear' or 'nearest'. No default.
+
+    Returns
+    -------
+    Returns a cube with the extracted point, and with adjusted
+    latitude and longitude coordinates.
+    """
```

Comment:
add a `Raises` section so the user knows there's an exc is raised if they enter mythical places :+1: